### PR TITLE
Burgermenu

### PR DESCRIPTION
**Pull request by FlorianBartier**
Added navigation bar with burger menu

### CODE REVIEW

**Review by hswiergiel — DISMISSED** [review]

File: frontend/src/components/BurgerMenu/BurgerMenu.jsx [hunk 1]
```diff
@@ -1,5 +1,94 @@
...
+            width="50vw"
+            isOpen={isOpen}
+            onOpen={handleIsOpen}
+            onClose={handleIsOpen}
+          >
+            <button
+              type="button"
+              onClick={() => {
+                closeSideBar();
+                // navigate("/");
```
Comment (LoicBrassart) [inline frontend/src/components/BurgerMenu/BurgerMenu.jsx]: Dead code (repeat until end of file)

File: frontend/src/components/BurgerMenu/style.jsx [hunk 2]
```diff
@@ -1,3 +1,119 @@
...
+    display: inline-block;
+    margin: 2.5rem;
+  }
+`;
+
+export const BurgerMenuContainer = styled.div`
+  @import url("https://fonts.googleapis.com/css2?family=Jost:wght@300&display=swap");
+  width: 100vw;
+  height: 60px;
+  /* box-shadow: 0 1px 3px #c91961; */
```
Comment (LoicBrassart) [inline frontend/src/components/BurgerMenu/style.jsx]: Dead code

File: package.json [hunk 3]
```diff
@@ -19,6 +19,9 @@
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
   },
+  "dependencies": {
+    "react-burger-menu": "^3.0.8"
+  },
```
Comment (LoicBrassart) [inline package.json]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils to fix
(@SM Plz read PR more carefully)

File: frontend/src/components/BurgerMenu/BurgerMenu.jsx (hunk 1 above)
Comment (FlorianBartier) [reply frontend/src/components/BurgerMenu/BurgerMenu.jsx]: problem solved

File: frontend/src/components/BurgerMenu/style.jsx (hunk 2 above)
Comment (FlorianBartier) [reply frontend/src/components/BurgerMenu/style.jsx]: problem solved

File: package.json (hunk 3 above)
Comment (FlorianBartier) [reply package.json]: problem solved

**Review by hswiergiel — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Better!

**Pull request merged**